<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEmergencyPatient</class>
 <widget class="QDialog" name="AddEmergencyPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #111111;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>30</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid #00d2ff;</string>
   </property>
   <property name="text">
    <string>Clinic Managemnet System - Emergency Patient</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: #1c1c1c;  
border: 1px solid #333333;  
border-radius: 6px;
color: #ffffff;            
padding: 10px 15px;
font-size: 15px;
font-weight: bold;
transition: 0.45s ease;
}

QPushButton:hover {
background-color: #111111; 
border: 1px solid #00d2ff; 
}

QPushButton:pressed {
    background-color: #000000;
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>150</y>
     <width>681</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: &quot;#1c1c1c&quot;;
border: none;
border-radius: 4px;
z-index: -1;
text-align: center;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>601</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="labelAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="verticalSpacing">
      <number>15</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Patient Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="patient_name">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Doctor's Name</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="doctors_name">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Gender</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="patient_gender">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Age</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="patient_age">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Disease</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="patient_disease">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    background-color: #1c1c1c;  
    border: 1px solid #333333;   
    border-radius: 4px;
    color: #ffffff;
    padding: 5px;
}
QLineEdit:focus {
    border: 1px solid #00d2ff;
    background-color: #222222;
}</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="add_patient_btn">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: #1c1c1c;  
border: 1px solid #333333;  
border-radius: 6px;
color: #ffffff;            
padding: 10px 15px;
font-size: 15px;
font-weight: bold;
transition: 0.45s ease;
}

QPushButton:hover {
background-color: #111111; 
border: 1px solid #00d2ff; 
}

QPushButton:pressed {
    background-color: #000000;
}</string>
       </property>
       <property name="text">
        <string>Add Patient</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
